include/mbedtls/psa_util.h

```diff
@@ -112,11 +112,16 @@ extern mbedtls_psa_drbg_context_t *const mbedtls_psa_random_state;
  *
  * \param grpid         An Mbed TLS elliptic curve identifier
  *                      (`MBEDTLS_ECP_DP_xxx`).
- * \param[out] bits     On success, the bit size of the curve.
+ * \param[out] bits     On success the bit size of the curve; 0 on failure.
  *
- * \return              The corresponding PSA elliptic curve identifier
+ * \return              On success the corresponding PSA elliptic curve identifier
  *                      (`PSA_ECC_FAMILY_xxx`).
- * \return              \c 0 on failure (\p grpid is not recognized).
+ * \return              \c 0 if \p grpid is not supported.
```

Comment:
This looks like the function returns 0 if the curve is not supported in the legacy API, but according to the note below, this is not true if the curve is supported in PSA.